hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestReplicationManager.java

```diff
@@ -394,6 +394,8 @@ public void testQuasiClosedContainerWithUnhealthyReplica()
     Assert.assertTrue(datanodeCommandHandler.received(
         SCMCommandProto.Type.deleteContainerCommand,
         replicaOne.getDatanodeDetails()));
+    Assert.assertEquals(currentDeleteCommandCount + 1,
```

Comment:
Can we add more test to check the other newly added metrices？

Reply:
Sure, I'll try my best to cover more metrics, maybe it is not so direct to simulate timeouts.